### PR TITLE
README Updates for CAM-ML

### LINKED ISSUE #5: Update CAM-ML documentation on the repo
### Issue Type

Documentation Update

### Issue Description

Add instructions in a README to:

- Describe project
- Describe how to point at this version of CAM from CIME
- Example namelist
- Example case setup

### Will this change answers?

No

### Will you be implementing this yourself?

Yes

### PR DESCRIPTION
**Pull request by jatkinson1000**
Closes #5

Update `README.md` with:

- details of the project
- how to set up CESM
- how to set up a testcase
- required modifications to the namelist to run the code

### CODE REVIEW

**Comment by jatkinson1000** [comment]
These instructions have been followed on Derecho using the `xml-updates` branch to successfully build.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,87 @@
-# CAM: The Community Atmosphere Model
+# CAM-ML: A Fork of The Community Atmosphere Model Implementing a Machine Learning Convection Parameterisation
+
+This Fork of the CAM model is a version in which we are aiming to implement a new
+convection parameterisation, YOG.
+The parameterisation is a machine learning implementation using a neural net trained
+on high-resolution cloud-resolving simulations in the SAM model as described in:\
+Yuval, J., O’Gorman, P.A.
+_Stable machine-learning parameterization of subgrid processes for climate modeling at
```
Comment (dorchard) [inline README.md]: Oh, I thought it was this paper: https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2020GL091363

File: README.md [hunk 2]
```diff
@@ -1,4 +1,87 @@
...
+tag.
+
+## Using this model in a CESM Run
+
+### Obtaining CESM
+
+Clone a copy of CESM from git and checkout the `cesm2.1.5` tag on which this work is based:
+```
+git clone https://github.com/escomp/cesm.git my_cesm_sandbox_2_1
+git checkout cesm2.1.5
```
Comment (dorchard) [inline README.md:27]: need to do `cd my_cesm_sandbox_2_1` in between these steps

Comment (jatkinson1000) [reply README.md:27]: Should be resolved in https://github.com/m2lines/CAM-ML/pull/6/commits/703cadd79d76c1dcc242d9ed8fbd991dece98d04

File: README.md (hunk 1 above)
Comment (jatkinson1000) [reply README.md]: Added in https://github.com/m2lines/CAM-ML/pull/6/commits/703cadd79d76c1dcc242d9ed8fbd991dece98d04

**Review by dorchard — APPROVED** [review]

**Pull request merged**